src/main/java/org/alfresco/rest/requests/FolderRules.java

```diff
@@ -110,6 +110,18 @@ public RestRuleModel createSingleRule(RestRuleModel ruleModel)
         return restWrapper.processModel(RestRuleModel.class, request);
     }
 
+    /**
+     * Create a rule link.
+     *
+     * @param parentRuleFolderId The parent folder with an existing rule.
+     * @return The id of the linked rule-set.
+     */
+    public RestRuleModel createRuleLink(String parentRuleFolderId)
```

Comment:
I think this method should be added in `org.alfresco.rest.requests.Node` class.
Moreover, I believe it should contain a new model class so that all would look more or less like this (not tested):
```
public RestRuleSetLinkModel createRuleLink(RestRuleSetLinkModel body) 
    {
        RestRequest request = RestRequest.requestWithBody(HttpMethod.POST, body, "nodes/{nodeId}/rule-set-links", repoModel.getNodeRef());
        return restWrapper.processModel(RestRuleSetLinkModel.class, request);
    }
```
However, due to the fact that [ACS-3446](https://alfresco.atlassian.net/browse/ACS-3446) is not done - there is no corresponding change in api-explorer project and thus we cannot use it to generate the proper model (named `RestRuleSetLinkModel` in the example above).
All in all - we need that additional model class for rule-set-link object (being the body of both request and response in this endpoint).

Reply:
On the first point, I initially had this method in the Node class due to the request being performed on a node, but since it's folder rules related I moved it to the the FolderRules class. Why do you think it needs to be in the Node class?

On the second point - I guess this ticket will be blocked until [ACS-3446](https://alfresco.atlassian.net/browse/ACS-3446) is done, or we can manually generate the model?